Python/Color_Picker/requirements.txt

```diff
@@ -0,0 +1,5 @@
+# Set Up
```

Comment:
Ummm. That's not what `requirements.txt`  is for.

Please have a look at this [blog](https://blog.usejournal.com/why-and-how-to-make-a-requirements-txt-f329c685181e) or any script which has a `requirements.txt` in this repo

Reply:
I will work on it.